tox.ini

```diff
@@ -11,6 +11,7 @@ deps =
   pytestlatest: pytest
   pytestmaster: git+https://github.com/pytest-dev/pytest.git@master
   pytestfeatures: git+https://github.com/pytest-dev/pytest.git@features
+  -etesting/foobar
```

Comment:
i  understand the need for this hack, please add a comment linking this fun hack with the test id